### PR TITLE
feat: add hero section to landing page

### PR DESCRIPTION
**Pull request by timDeHof**
having trouble with `getServerSideProps` to work. keep getting a syntax error from it
Screenshot 2023-03-28 at 4 29 02 PM

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-46)

File: pages/index.jsx [hunk 1]
```diff
@@ -36,3 +41,16 @@ const IssuesPage = () => {
 };
 
 export default IssuesPage;
+
+export const getServerSideProps: GetServerSideProps<{
```
Comment (timDeHof) [inline pages/index.jsx:45]: its gives a syntax error on data part in the `getServerSideProps` method. but I followed Next.js docs on it.

Comment (timDeHof) [reply pages/index.jsx]: changed to getStaticProps after research

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-46)

**Pull request merged**